File: prak2 c++/Product.cpp
#include <iostream>
#include <string>

using std::string;
using std::cout;
using std::endl;

class Product{

    private:
        int price;
        string idProduct;

    public:
        Product(){
            //konstruktor
        }

        Product(int harga, string id){
            //konstruktor
            price = harga;
            idProduct = id;
        }


        //set
        void setPrice(int harga){
            price = harga;
        }

        void setIdProduct(string id){
            idProduct = id;
        }

        //get
        int getPrice(){
            return price;
        }

        string getIdProduct(){
            return idProduct;
        }


        //methods
        void tampilkanProduct(){
            cout << "Price : " << getPrice() << endl;
            cout << "idProduct : " << getIdProduct() << endl;
        }


        ~Product(){
            //destruktor
        }

};

File: prak2 python/Product.py
class Product:

    def __init__(self, price, id):
    #konstruktor
        self.price = price
        self.idProduct = id

    #set
    def setPrice(self, price):
        self.price = price

    def setIdProduct(self, idProduct):
        self.idProduct = idProduct

    #get
    def getPrice(self):
        return self.price

    def getIdProduct(self):
        return self.idProduct

    #methods
    def tampilkanProduct(self):
        print("Price : " + str(self.getPrice()))
        print("idProduct : " + self.getIdProduct())

